docs/yaml/builtins/meson.yaml

```diff
@@ -88,7 +88,7 @@ methods:
 
     posargs:
       script_name:
-        type: str | file | external_program | exe | custom_tgt | custom_idx
+        type: str | file | external_program | exe
```

Comment:
This seems incorrect. Generating a custom script to run seems like a thing we should support and the comment below this even explicitly says that custom targets are accepted.

Reply:
I tested it, and meson throws an error in this case.  Maybe the actual type annotation needs updating?

Reply:
`add_install_script` should accept `CustomTarget` and `CustomTargetIndex`, if it doesn't that's a bug (and probably my fault :/)